tools/ros2_dependencies.repos

```diff
@@ -1,13 +1,5 @@
 repositories:
```

Comment:
Are these removed because they're on released binaries now?

Reply:
Yes, you're right.